docs/source/2019Barry.rst

```diff
@@ -120,6 +146,28 @@ Hurricane Track and Intensity
     * The time variation of maximum wind speed in GFS.v16.0.10 agrees well with Best Track.
     * The minimum sea level pressure reaches to a lower value in GFS.v16.0.10 (982 hPa) compared with Best Track (993 hPa). 
 
+  .. group-tab:: SRW.v1.0
+
+    .. figure:: images/2019Barry/tracker_Barry_srwv1.png
+      :width: 400
+      :align: center
+
+      Hurricane tracks from SRW_RRFSv1alpha (blue line), SRW_GFSv15p2 (purple dash line), MRW_GFSv15p2 (red line), and Best Track (black line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt). 
+
+    * Both MRW_GFSv15p2 and SRW_GFSv15p2 generate right-of-track biases. 
+    * Hurricane track and intensity simulated by SRW_RRFSv1apha are closer to Best Track compared with SRW_GFSv15p2 and MRW_GFSv15p2.
+
+
+    .. figure:: images/2019Barry/tracker_ws_mslp_Barry_srwv1.png
+      :width: 1200
+      :align: center
+
+      Time series of the vortex maximum surface wind speed (WS, left panel) and minimum mean sea level pressure (MSLP, right panel)
+
+    * The peak wind speeds at the vortex center in MRW_GFSv15p2 (58 kts) SRW_RRFSv1alpha (58 kts) are closer to Best Track (67 kts) compared with SRW_GFSv15p2 (52 kts).
```

Comment:
"and" missing between "(58 kts)" and "SRW_RRFSv1alpha"

Reply:
Here is the directory for the data on Hera, and the total size is around 3.2 TB. Thanks! @climbfuji 
hfe12:/scratch2/BMC/fv3lam/lpan/UFS_SRW_app_data>ls */*.gz
2018100900/2018100900_bc.gfs.nemsio.tar.gz  2019092512/2019092512.gfs.nemsio.tar.gz	2020040912/2020040912_bc.gfs.nemsio.tar.gz
2018100900/2018100900.gfs.nemsio.tar.gz     2019102812/2019102812_bc.gfs.nemsio.tar.gz	2020040912/2020040912.gfs.nemsio.tar.gz
2019052300/2019052300_bc.gfs.nemsio.tar.gz  2019102812/2019102812.gfs.nemsio.tar.gz	2020042912/2020042912_bc.gfs.nemsio.tar.gz
2019052300/2019052300.gfs.nemsio.tar.gz     2020011812/2020011812_bc.gfs.nemsio.tar.gz	2020042912/2020042912.gfs.nemsio.tar.gz
2019071200/2019071200_bc.gfs.nemsio.tar.gz  2020011812/2020011812.gfs.nemsio.tar.gz	2020072300/2020072300_bc.gfs.nemsio.tar.gz
2019071200/2019071200.gfs.nemsio.tar.gz     2020020312/2020020312_bc.gfs.nemsio.tar.gz	2020072300/2020072300.gfs.nemsio.tar.gz
2019092512/2019092512_bc.gfs.nemsio.tar.gz  2020020312/2020020312.gfs.nemsio.tar.gz

Reply:
Thanks, @panll - will upload those today and let you know when it is finished.

Reply:
@panll uploading this amount of data is VERY slow. I'll let you know when I think it is done, then you can check if all necessary files are there.

Reply:
Thanks, Dom! Yes, the data are huge (3.2TB). Jun seems don't have a better way to reduce the nemsio file size. @climbfuji